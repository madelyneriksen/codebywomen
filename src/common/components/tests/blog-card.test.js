import React from 'react';
import renderer from 'react-test-renderer';
import BlogCard from '../blog-card';


describe("Navbar", () => {
  it("Matches Snapshot", () => {
    const data = {
      title: "The Zen of Programming",
      slug: "/zen-of-programming",
      author: "Ali Spittel",
      tags: ['javascript', 'python'],
      description: "The zen of programming",
    }
    const tree = renderer.create(<BlogCard props={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
