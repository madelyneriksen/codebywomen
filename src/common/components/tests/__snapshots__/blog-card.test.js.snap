// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Matches Snapshot 1`] = `
<div
  className="bg-white bl bw2 br2 b--blue pa3 shadow-5 w-100 mv2"
>
  <a
    className="db f4 sans-serif mv1 navy"
  />
  <span
    className="db f5 sans-serif gray"
  >
    by 
  </span>
  <p
    className="db f5 sans-serif lh-copy"
  />
  <div
    className="flex flex-wrap justify-start"
  />
</div>
`;
